src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EVP.cs

```diff
@@ -71,6 +80,31 @@ internal static partial class Crypto
             }
         }
 
+        internal static unsafe int EvpDigestFinalXOF(SafeEvpMdCtxHandle ctx, Span<byte> destination)
+        {
+            fixed (byte* pDestination = destination)
```

Comment:
AFAIK you can pass `Span<byte>` to intro and the `fixed` would be generated automatically. But I think it is ok as follows existing pattern.

Reply:
Subsequently it'd be nice to clean them all up, now that span-based marshaling is supported.

Reply:
I cleaned up this file, I'll make a note to track cleaning up other p/invokes elsewhere. I didn't want to lump a large cleanup effort in to this PR, but figured we might as well do the right thing going forward, at least.